src/components/Payments/AddPage.js

```diff
@@ -1,114 +1,91 @@
 import React, { useCallback } from "react";
-import ReactDOM from "react-dom";
 import { Finance24 } from "@carbon/icons-react";
 import styled from "styled-components";
 import { GridThreeUp } from "../pageStyles";
 import PageLayout from "../PageLayout";
 import CardButton from "../CardButton";
-import Button from "../Button";
 import Breadcrumbs from "../Breadcrumbs";
 import chime from "../../assets/images/chime.png";
+import paypal from "../../assets/images/paypal.png";
 import { usePlaidLink } from "react-plaid-link";
 import { AppContext } from "../../api/AppContext";
 import { useHistory } from "react-router-dom";
+import { spacing } from '../../theme';
 
-const { REACT_APP_PLAID_KEY, REACT_APP_PAYMENT_TYPE } = process.env;
+const { REACT_APP_PLAID_KEY } = process.env;
 
-const AACComponent = window.paypal
-  ? window.paypal.PayoutsAAC.driver("react", {
-      React,
-      ReactDOM,
-    })
-  : null;
+const Details = styled.p`
+  margin-bottom: ${spacing[5]};
+`;
 
-const Chime = styled.img`
+const CardIcon = styled.img`
   width: 24px;
   height: 24px;
 `;
 
 export default () => {
   const history = useHistory();
-  const { api, fetchUser } = React.useContext(AppContext);
+  const { user, api, fetchUser } = React.useContext(AppContext);
   const onSuccess = useCallback(async (token, metadata) => {
-    await api.setToken(token, metadata.account_id);
+    await api.setStripeToken(token, metadata.account_id);
     await fetchUser();
     history.push("/payments");
   }, []);
-  const onPaypal = useCallback((token) => {
-    // api.setTokenPaypal(token)
-  }, []);
   const config = {
     clientName: "BlockPower",
+    // TODO: Shouldn't this be an env var?
     env: "sandbox",
```

Comment:
Anyone know is this is a bug?

Reply:
I don't know where that is supposed to display clientName but I don't think it's an issue @cooperka

Reply:
To be clear I'm wondering why Plaid is being used in "sandbox" mode. It may suppress errors or do something weird in production. Just a thought!

Reply:
@brodavi - is this a concern or we have the right PayPal environment set in env vars on production?

Reply:
talked with David, there are definitely lots of skeletons on the frontend that should be env vars including this one.  It would be great to clean some of that up, we'll see if we can make time for it